app/models/concerns/notificable.rb

```diff
@@ -9,6 +9,11 @@ def last_notification
     notifications.max_by(&:created_at)
   end
 
+  def last_convocation_by(format)
+    # Pour afficher les infos de delivrance des convocations pour les rdv créés uniquement
+    notifications.where(format: format, event: "participation_created").max_by(&:created_at)
```

Comment:
Pour éviter de potentielles N+1 on peut itérer sur l'array (comme on le fait dans le concern `Invitable`) de la sorte: 

```suggestion 
    # Pour afficher les infos de delivrance des convocations pour les rdv créés uniquement
   convocations.select { |convocation| convocation.format == format }.max_by(&:created_at)
```

Reply:
Effectivement 👍  merci